sidekick_core/lib/sidekick_core.dart

```diff
@@ -197,23 +197,29 @@ class SidekickCommandRunner<T> extends CommandRunner<T> {
         Repository.requiredSidekickPackage,
         ['dependencies', 'sidekick_core'],
       );
+      if (sidekickCoreVersion == null) {
+        // Couldn't parse sidekick_core version. Most likely because it uses
```

Comment:
We could fallback to `VersionChecker.getResolvedVersion` because `pubspec.lock` contains version information for git or path dependencies as well. But that's probably overkill, so you can ignore this comment

Reply:
Good idea for the future 👍